src/stories/contrib/debug_toolbars/flask/panels.py

```diff
@@ -0,0 +1,65 @@
+"""
+stories.contrib.debug_toolbars.flask.panels
+-------------------------------------------
+
+This module contains integration with flask-debugtoolbar.
+
+:copyright: (c) 2018-2019 dry-python team.
+:license: BSD, see LICENSE for more details.
+"""
+
+import stories._context
+from flask_debugtoolbar.panels import DebugPanel
+from flask import render_template
+
+
+original_context_init = stories._context.Context.__init__
+
+def track_context(storage):
+    def wrapper(ctx, *args, **kwargs):
+        original_context_init(ctx, *args, **kwargs)
+        storage.append(ctx)
+
+    return wrapper
+
+def pluralize(number, singular, plural=None):
+    if plural is None:
+        plural = f"{singular}s"
+    if number == 1:
+        return f"{number} {singular}"
```

Comment:
f-strings are great and I use it a lot personally. But it will fail on Python 2 and early Python 3 versions. If you do not mind, can you change them to `%` format? 😬

Reply:
Good point, on it :)